src/api/web.ts

```diff
@@ -232,3 +232,12 @@ export function uploadAvatar(req: express.Request | any, res: express.Response):
     .then(() => res.status(200).json({ data: webPath }))
     .catch(err => res.status(200).json({ message: err }));
 }
+
+export function userProfile(req: express.Request, res: express.Response): any {
+  let username = req.query.username;
+  let auth = getAuth();
+  let user = auth.users.find(user => user.name === username);
+  delete user.password;
```

Comment:
we don't want to send password hash along with the response... that could cause potential security concerns.